src/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -96,6 +98,21 @@ testRule(rule, {
   }, {
     code: "a { font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', sans-serif; }",
     message: messages.rejected("BlinkMacSystemFont"),
+  }, {
+    code: "a { font: italic 300 16px/30px Arial, serif; }",
+    message: messages.expected("Arial"),
+    line: 1,
+    column: 32,
+  }, {
+    code: "a { font: italic 1000 16px/30px Arial, serif; }",
+    message: messages.expected("1000 Arial"),
```

Comment:
Custom font-name is just a string as per spec it can be anything right.  What if font is named as 1983